MAINTAINERS

```diff
@@ -96,7 +96,10 @@ S: Maintained
 F: xlators/features/changelog/
 
 Distributed Hashing Table (DHT)
-P: Susant Palai <spalai@redhat.com>
+M: Mohit Agrawal <moagrawa@redhat.com>
+M: Barak Sason Rofman <bsasonro@redhat.com>
```

Comment:
Just curious, shouldn't new contributors be added as Peers and then graduate as Maintainers with consistent contributions to the component?

Reply:
If the team believe that Peer status is more appropriate, I don't object it.

To me the main reasons for this change were:
1 - Update the doc to reflect the current status of the people involved.
2 - Allow the DHT team to be more independent, as prior to this change none of the team members had merge rights (and following the migration to GitHub, no rights to run regression tests as well) which adversely affect workflow as we need the assistance of other maintainer in order to perform these tasks.

Reply:
Valid points, both! @itisravi considering we didn't had any one with active contributors in this list, it is fine in IMO.

With triggering regression issues, its crucial for update!